persist/src/main/java/com.codeforcommunity/database/tableImpl/StubCommentTableImpl.java

```diff
@@ -0,0 +1,93 @@
+package com.codeforcommunity.database.tableImpl;
+
+import com.codeforcommunity.database.records.CommentRecord;
+import com.codeforcommunity.database.seeder.Seeder;
+import com.codeforcommunity.database.table.ICommentTable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Our implementation of the {@link ICommentTable} in our database. This class will eventually be
+ * replaced once we start actually working with the database, but we'll still use it for testing
+ * purposes.
+ */
+public class StubCommentTableImpl implements ICommentTable {
+  // A map of an integer (representing a postId) to a Map of that post's comment id to comment.
+  private final Map<Integer, Map<Integer, CommentRecord>> commentMap;
+
+  public StubCommentTableImpl() {
+    this.commentMap = new HashMap<>();
+  }
+
+  @Override
+  public List<CommentRecord> getByPostId(int postId) {
+    Map<Integer, CommentRecord> commentMap = this.commentMap.getOrDefault(postId, new HashMap<>());
+    return List.copyOf(commentMap.values());
```

Comment:
guessing we are assuming a solid understanding of Java

Reply:
I've added comments in #4 since that's where it first appears.